native/exe/SageLauncher/launcher.cpp

```diff
@@ -637,8 +654,12 @@ int WINAPI WinMain( HINSTANCE hInst, 	/*Win32 entry-point routine */
 			}
 		}
 	}
-	chdir(folderPath);
-	delete [] folderPath;
+	errno_t err;
+	if ((err = _chdir(folderPath)) != 0) // If this doesn't return 0 we have bigger problems
+	{
+		delete[] &folderPath;

```

Comment:
This should just be 'delete [] folderPath'; Get rid of the ampersand.

Reply:
Better yet..change folderPath to be non-heap allocated (i.e. char folderPath[2048]; and use sizeof(folderPath) in place of 2048 in the GetModuleFilename call).